modules/ROOT/pages/_partials/include-deploy-resource-allocation.adoc

```diff
@@ -0,0 +1,201 @@
+
+* 0.7 GiB memory for Mule 4
+* 0.5 GiB memory for Mule 3
+
+When Anypoint Monitoring is enabled, 50MB memory is reserved for running the Anypoint Monitoring sidecar containers, which can burst up to 70MB for each application deployed when processing metrics data.
+// end::memoryAllocation[]
+// tag::calculatingMemory[]
+== Calculating Memory Allocation on CloudHub and Runtime Fabric
+
+Anypoint Platform allocates native and heap memory for a deployed application. Heap memory is the portion of the total memory that is made available to the Mule runtime and the application. Heap memory is used for tasks like handling payload.
+
+Both CloudHub and Anypoint Runtime Fabric allocate both types of memory. However, they differ in how the memory allocation of each memory type is calculated.
+
+* Runtime Fabric lists the total memory available for an application. +
+The available heap memory is approximately half of the total memory allocated to an application.
+* CloudHub describes minimum memory requirements in terms of the heap memory available to an application.
+// end::calculatingMemory[]
+// tag::startupTimes[]
+== Application Startup Times
+
+The startup time for a Mule application is correlated to the total number of vCPU cores the application has access to. The following lists approximate startup times for a simple Mule proxy application performing processing on a payload:
+
+[%header,cols="2*a"]
+|===
+| vCPU Cores | Approximate Startup Time
+| `1.00` | Less than 1 minute
+| `0.50` | Under 2 minutes
+| `0.10` | 6 to 8 minutes
+| `0.07` | 10 to 14 minutes
+|===
+// end::startupTimes[]
+// tag::applicationPerformance[]
+== Application Performance
+
+The resources allocated to your Mule application determine the application's performance. The following table lists approximate values for throughput based on the total number of vCPU cores allocated for a single Mule application performing simple processing on a 10-KB payload:
+
+[%header,cols="3*a"]
+|===
+| vCPU Cores | Concurrent Connections | Avg Response Time (ms)
+| `1.00` | 10 | 15
+| `0.50` | 5 | 15
+| `0.10` | 1 | 25
+| `0.07` | 1 | 78
+|===
+
+[NOTE]
+Run performance and load testing on your Mule applications to determine the number of resources to allocate.
+
+// end::applicationPerformance[]
+// tag::loadBalancerMemoryAllocation[]
+== Internal Load Balancer Memory Allocation
+
+Internal load balancer memory requirements are impacted by the number of threads, response time latency, and message sizes. Use the following guidelines when allocating memory:
+
+* .5 GB (Default): For fewer than 500 simultaneous active connections.
+* 1.5 GB (Large): For one or both of the following scenarios:
+
+** 500 or more simultaneous active connections.
+** Security policies are enabled.
+
+[NOTE]
+These are general guidelines, and individual environments might require adjustments.
+
+// end::loadBalancerMemoryAllocation[]
+// tag::internalLoadBalancer[]
+== Internal Load Balancer
```

Comment:
Internal Load Balancer (Edge) is only applicable on gravitational appliances. Does not apply to BYOK

Reply:
ok